1-exercises/B-object-arrays/exercise-2.js

```diff
@@ -16,7 +16,7 @@ let store = [
 
 let money = 1.5;
 
-let thingsICanBuy =
+let thingsICanBuy =store.filter((thingsAffored)=>thingsAffored.price < 1.5).map((thingsAffored) => thingsAffored.item)
```

Comment:
Please do keep in mind that there should be space in between, as this makes the code easier to read. It wont change the functionality of the code it will just make the work easier of others when they are reading your code.
```suggestion
let thingsICanBuy = store.filter((thingsAffored) => thingsAffored.price < 1.5).map((thingsAffored) => thingsAffored.item)
```